src/wp-includes/functions.php

```diff
@@ -8071,8 +8071,8 @@ function wp_cache_set_last_changed( $group ) {
 	 * @since 6.3.0
 	 *
 	 * @param string    $group         The cache group name.
-	 * @param int       $time          The new last changed time.
-	 * @param int|false $previous_time The previous last changed time. False if not previously set.
+	 * @param string    $time          The new last changed time (msec sec).
```

Comment:
Minor point: The `$group` and `$time` prams should be indented further so they line up with `$previous_time`.

Reply:
Done!